test/cases/configuration/test_configuration.py

```diff
@@ -51,90 +48,93 @@ def test_in_http(self):
         #     datadog_environment fooment;
         #     datadog_agent_url http://bogus:1234;
         #     datadog_propagation_styles B3 Datadog;
-        expected['defaults']['service'] = 'foosvc'
-        expected['defaults']['environment'] = 'fooment'
-        expected['collector']['config'][
-            'url'] = 'http://bogus:1234/v0.4/traces'
-        styles = ['B3', 'Datadog']
-        expected['injection_styles'] = styles
-        expected['extraction_styles'] = styles
-
+        del config["runtime_id"]
+        del expected["runtime_id"]
+        expected["defaults"]["service"] = "foosvc"
+        expected["defaults"]["environment"] = "fooment"
+        expected["collector"]["config"]["traces_url"] = "http://bogus:1234/v0.4/traces"
+        expected["collector"]["config"][
+            "telemetry_url"
+        ] = "http://bogus:1234/telemetry/proxy/api/v2/apmtelemetry"
```

Comment:
That's the part I needed to change. Once `dd-trace-cpp` will use the latest version, @dgoffredo you will need to port those changes.